spec/openapi.yaml

```diff
@@ -53,182 +156,498 @@ paths:
           format: int64
       responses:
         200:
-          description: Resource response successful
+          description: A specific resource's details
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Resource'
-              examples:
-                0:
-                  value: |
-                    {"apiVersion":"1.0","data":{"category":"Books","created_at":"2019-06-15 17:55:32","downvotes":0,"id":3,"languages":[],"last_updated":"","name":"Free Tech Books","notes":"Focuses on general computer science concepts rather than a specific language","paid":false,"times_clicked":0,"upvotes":0,"url":"http://www.freetechbooks.com/"},"status":"ok","status_code":200}
+              example: |
+                {
+                  'apiVersion':'1.0',
+                  'data':{
+                    'category':'Books',
+                    'created_at':'2019-06-15 17:55:32',
+                    'downvotes':0,
+                    'id':3,
+                    'languages':[],
+                    'last_updated':',
+                    'name':'Free Tech Books',
+                    'notes':'Focuses on general computer science concepts rather than a specific language',
+                    'paid':false,
+                    'times_clicked':0,
+                    'upvotes':0,
+                    'url':'http://www.freetechbooks.com/'
+                    
+                  },
+                  'status':'ok',
+                  'status_code':200
+                }
+        404:
+          $ref: '#/components/responses/NotFound'
+    put:
+      summary: Update Resource
+      description: Updates an existing learning resource entry. The request body should be a JSON object containing the properties to be updated and their corresponding new values. 
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
```

Comment:
I believe this should have `$ref: '#/components/schemas/Resource_data'`. It needs to reflect the same 6 valid options for a request as the `Create Resource` endpoint